lib/src/actions/actions.dart

```diff
@@ -384,6 +400,9 @@ abstract class SubstrandNameSet
 
   @memoized
   int get hashCode;
```

Comment:
I forget how this works with Domains and Loopouts, but if the action also indicates whether it was a Domain or Loopout, it would be good to indicate that here (i.e., somehow have this message say "set domain name" or "set loopout name", based on auxiliary information). The notion of a "Substrand" as a parent type is sort of an internal implementation detail of scadnano, rather than something we expect the web interface user to know about.

Reply:
https://github.com/UC-Davis-molecular-computing/scadnano/pull/752/commits/839c29f2b4c3d05838160b9eaba07e04e5b1e015